### DISCUSSION TITLE
static import intellisense

### DISCUSSION THREAD

**Discussion by rafaeltuelho** [redhat-developer/vscode-java#2744] [Q&A] [answered]
I'm having. hard time with it comes to adding a `static` `import`. I noticed that the java extension struggles to suggest static members in the IntelliSense.

![image](https://user-images.githubusercontent.com/39063/196670403-40121bfe-9d5a-4a8f-b8fc-8fc108ce7749.png)

It's a pain to go to the top and add the import manually.

Is it a known issue?

**Comment by rafaeltuelho** [comment, marked as answer]
Okay, found this explanation https://github.com/redhat-developer/vscode-java/issues/796#issuecomment-473133024

**Reply by fbricon** [reply]
Indeed you need to amend the java.completion.favoriteStaticMembers list. Which assertion library are you using?

**Reply by rafaeltuelho** [reply]
Thanks for confirming @fbricon 
I'm using Hamcrest in this case.
